test-migration.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# make -C tools start-db
+
+psql -h localhost -U postgres -d postgres <<EOF
```

Comment:
do you want to add something like this?

Reply:
I'll take this out when everything is done. I don't think we keep migration tests anywhere in the repo do we?